libs/ktem/ktem/reasoning/simple.py

```diff
@@ -711,7 +831,22 @@ def get_pipeline(cls, settings, states, retrievers):
             retrievers: the retrievers to use
         """
         prefix = f"reasoning.options.{cls.get_info()['id']}"
-        pipeline = cls(retrievers=retrievers)
+        example_path = (
+            Path(getattr(flowsettings, "KH_APP_DATA_DIR", ""))
+            / "rephrase_question_train.json"
+        )
```

Comment:
Please avoid hard-coding inside class definitions. The `rephrase_question_train.json` is not guaranteed to exist.

Note that `KH_APP_DATA_DIR` is intended for the app's data that is generated during runtime and not for storing configuration data.